### ISSUE TITLE
[Question] See the actual kubectl command its running?

### ISSUE THREAD

**Issue by leninhasda** [Azure/vscode-kubernetes-tools#648]
Is there a way to see the actual kubectl command that it's running? Maybe in output or debug console ?

**Comment by itowlson** [comment]
I don't think we do this at the moment.  Some commands get sent to the Output pane but most are issued silently.  It would be good to have at least an option for this, though the places where we invoke `kubectl` are rather disorganised at the moment so I'll need to figure out where to add this logging.  Thanks for the suggestion!

**Comment by aelij** [comment]
In addition, it would be helpful to see the output of kubectl in case of failure.
